### PR TITLE
Add v4l2/logging camera troubleshooting info

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

**Review by gerth2 — DISMISSED** [review]

File: source/docs/troubleshooting/camera-troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+| VisionSource created             | Adding 1 configs to VMM.                               | No output from VisionSourceManager |
++----------------------------------+--------------------------------------------------------+------------------------------------+
+
+Determining detected cameras in Video4Linux (v4l2)
+--------------------------------------------------
+
+On Linux devices (including Raspberry Pi), PhotonVision uses WPILib's CSCore to interact with video devices, which internally uses Video4Linux (v4l2). CSCore, and therefore Photon, requires that cameras attached have good v4l drivers for proper functionality. These should be built into the Linux kernel, and do not need to be installed manually. Valid picamera setup (from /boot/config.txt) can also be determined using these steps. The list-devices command will show all valid video devices detected, and list-formats the list of "video modes" each camera can be in.
+
+- For picams: edit the config.txt file as described in the :ref:`picam configuration page <docs/hardware/picamconfig:pi camera configuration>`
+- SSH into your Pi: :code:`ssh pi@photonvision.local` and enter the username "pi"/password "raspberry"
```
Comment (mdurrani808) [inline source/docs/troubleshooting/camera-troubleshooting.rst:42]: just say username "pi" and password "raspberry"
no point in a slash

**Review by mdurrani808 — DISMISSED** [review]
good stuff

File: source/docs/troubleshooting/camera-troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+- run :code:`v4l2-ctl --list-devices` and :code:`v4l2-ctl --list-formats`
+
+We expect an output similar to the following. For picameras, note the "unicam" entry with path :code:`platform:3f801000.csi` (if we don't see this, that's bad), and a huge list of valid video formats. USB cameras should show up similarly in the output of these commands.
+
+.. tab-set-code::
+  .. code-block:: Working
```
Comment (mdurrani808) [inline source/docs/troubleshooting/camera-troubleshooting.rst:48]: Not a valid lexer. List here: https://pygments.org/languages/

File: source/docs/troubleshooting/camera-troubleshooting.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        [1]: 'UYVY' (UYVY 4:2:2)
+        [2]: 'YVYU' (YVYU 4:2:2)
+        [3]: 'VYUY' (VYUY 4:2:2)
+        <snip>
+        [42]: 'Y12P' (12-bit Greyscale (MIPI Packed))
+        [43]: 'Y12 ' (12-bit Greyscale)
+        [44]: 'Y14P' (14-bit Greyscale (MIPI Packed))
+        [45]: 'Y14 ' (14-bit Greyscale)
+
+  .. code-block:: Broken
```
Comment (mdurrani808) [inline source/docs/troubleshooting/camera-troubleshooting.rst:89]: See above

File: source/docs/troubleshooting/camera-troubleshooting.rst (hunk 2 above)
Comment (mcm001) [reply source/docs/troubleshooting/camera-troubleshooting.rst:48]: I wanted to show a tab group with "working" and "not" in the title, though. How could I do that?

**Review by mdurrani808 — APPROVED** [review]

**Pull request merged**